src/app/multiple-records/multiple-records.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="govuk-width-container">
+  <a [routerLink]="['/']" class="govuk-back-link">Back</a>
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row govuk-!-margin-bottom-3">
+      <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">
+          Select a technical record
+        </h1>
+        <p class="govuk-body govuk-!-font-size-19">
+          Multiple technical records found matching this search
+        </p>
+      </div>
+    </div>
+
+    <ng-container *ngFor="let techRecord of techRecords | FilterMultipleRecords; let i = index">
+      <ng-container *ngIf="techRecord.techRecord as activeRecord">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-one-half">
+            <h3 class="govuk-heading-m">Technical record {{ i + 1 }}</h3>
+          </div>
+          <div class="govuk-grid-column-one-half">
+            <a class="govuk-link govuk-!-font-size-19" [routerLink]="['/technical-record', i]">Select technical record</a>
```

Comment:
using the index is not ideal. Is either we have a genuine identifier or not. Think about how searchable or bookmark this can be